docs/infrastructure_and_maintenance/cache/http_cache/content_aware_cache.md

```diff
@@ -228,15 +228,15 @@ For full content tagging when inline rendering, use the following:
 {{ ibexa_http_cache_tag_location(location) }}
 ```
 
-2\. `ez_http_cache_tag_relation_ids()` or `ez_http_cache_tag_relation_location_ids()`
+2\. `ibexa_http_cache_tag_relation_ids()` or `ibexa_http_cache_tag_relation_location_ids()`
 
 When you want to reduce the amount of tags, or the inline content is rendered using ESI, a minimum set of tags can be set:
 
 ``` html+twig
-{{ ez_http_cache_tag_relation_ids(content.id) }}
+{{ ibexa_http_cache_tag_relation_ids(content.id) }}
 
 {# Or using array for several values #}
```

Comment:
Should this comment line be displayed too in the output? If yes, it's not rendered in the readthedocs.

Reply:
@juskora It won't be fixable in this PR. The problem is at a higher level. You can put `{# … #}` where ever in a Markdown, it will be hidden.

Markdown:
```md
?{# hidden #}!
```
Rendering:
```html
?!
```